eessi-2023.06-eb-4.7.2-2021b.yml

```diff
@@ -1,2 +1,3 @@
 easyconfigs:
   - GCC-11.2.0
+  - foss-2021b
```

Comment:
We can try trigger this just to see how things work, but it won't fully build without problems.

For CMake, we need to use https://github.com/easybuilders/easybuild-easyblocks/pull/2248.
There's two CMake easyconfigs that use `GCCcore/11.2.0` as toolchain, and both are used, so we should install both with this easyblock PR:

```yaml
  - CMake-3.21.1-GCCcore-11.2.0.eb:
      options:
        include-easyblocks-from-pr: 2248
  - CMake-3.22.1-GCCcore-11.2.0.eb:
      options:
        include-easyblocks-from-pr: 2248
```

For OpenBLAS on `aarch64/*`, we will probably need to relax things a bit w.r.t. LAPACK tests by increasing the value for `max_failing_lapack_tests_num_errors` via a hook, because we noticed that slightly more LAPACK tests are failing with numerical errors on Arm 64-bit targets (which should be OK to ignore for now).

So, maybe start with only `CMake` as a first step?